feed/templates/memberships/list.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'base.html' %}
+
+% block content %}
+<div class="container">
+<br>
+<h1>Choose a plan according to your needs</h1>
+<hr>
+<div class="row">
+{% for object in object_list %}
+<div class="col-sm-4 col-md-4">
+<div class="card" style="width: 18rem;">
+<!-- <img class="card-img-top " src="{{ object.image.url }}" alt="Card image cap"> -->
+<div class="card-body">
+<h5 class="card-title">{{ object.membership_type }}</h5>
+<p class="card-text">{{ object.desc | linebreaks }}</p>
```

Comment:
This is a non-blocking comment: but indentation can go a long way for readability ;)